index.html

```diff
@@ -104,3 +104,4 @@ <h2>Heading</h2>
         </script>
     </body>
 </html>
+<!-- Add forgot password reset options-->
```

Comment:
I cannot find any text or links that provide information to help users who have forgotten their password?